### ISSUE TITLE
retrain deepForest on hiperspectral images

### ISSUE THREAD

**Issue by MarconiS** [weecology/DeepForest#96]
Using automatically classified species labels and current results of deepForest on rgb data, pretrain on few million tree crowns on hiperspectral clips to simultaneously detect, segment and predict species labels from NEON data.

Tackling this issue will require:

- [ ] tweaking keras_retinanet to be able to use multiple channels
- [ ] retrain from scratch the model, since imagined weights may not be informative in this case
- [ ] an extreme dose of luck to have the model even learning something :) 

Will be starting with 1 relatively hard site (HARV)
